src/components/categoriesBar/CategoriesBar.jsx

```diff
@@ -1,8 +1,47 @@
-import React from "react";
+import React, { useState } from "react";
 import "./_categoriesBar.scss";
 
+const keywords = [
```

Comment:
Can this be created in a separate constant file

Reply:
@faran4engg i think this will be removed later I will come from API